marketplace/dev/Dockerfile

```diff
@@ -31,7 +31,8 @@ RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - && \
 
 COPY marketplace/deployer_util/* /bin/
 COPY scripts/* /scripts/
-COPY marketplace/dev/setup_kubeconfig.sh /bin/setup_kubeconfig.sh
+COPY marketplace/dev/*.sh /bin/
```

Comment:
nit: Could we move `marketplace/dev/setup_config.sh ` to `marketplace/dev/bin/` and change this instruction to `COPY marketplace/dev/bin/* /bin/`?

Reply:
Done.

Reply:
Thanks!